public/css/manager.css

```diff
@@ -11741,45 +11741,31 @@ body {
           box-shadow: 0 0 0 3px rgba(255, 255, 255, 0.25);
 }
 
-.news-card{
-    margin: 20px 0;
-}
-
-.news-card2{
-  padding:15px 20px; 
-}
 
 .news-head{
   display: flex; 
   align-items: center; 
 }
 
-.news-option{
-  margin: 0 10px;
-}
-
 .news-span{
   height: 25px;
   width: 25px;
 }
 
-.perfil-image{
-  width:70px; height:70px; border-radius:150px;border:1px solid #666;
-}
-
-.news-item{
-  margin: 10px 10px;
-}
-
 .carousel-size{
   height: 400px;
 }
 
-.news-title{
+.create-news-container{
```

Comment:
all the css class should be in the SASS file, if have any questions please contact me 🤓